src/Uno.UWP/ApplicationModel/Contacts/ContactPicker.Android.cs

```diff
@@ -0,0 +1,384 @@
+﻿#nullable enable
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Android.App;
+using Android.Content;
+using Android.Database;
+using Android.Provider;
+using Uno.Helpers.Activities;
+using Uno.UI;
+using Windows.Extensions;
+
+namespace Windows.ApplicationModel.Contacts
+{
+	public partial class ContactPicker
+	{
+		private static Task<bool> IsSupportedTaskAsync() => Task.FromResult(true);
+
+		private async Task<Contact[]> PickContactsAsync(bool multiple, CancellationToken token)
+		{
+			if (!await PermissionsHelper.CheckReadContactsPermission(default))
+			{
+				if (!await PermissionsHelper.TryGetReadContactsPermission(default))
+				{
+					throw new InvalidOperationException("android.permission.READ_CONTACTS permission is required");
+				}
+			}
+
+			if (ContextHelper.Current == null)
```

Comment:
Does the ApplicationContext is enough? (We try to avoid the usage of the `ContextHelper`)

Reply:
I missed that, when did we start trying to avoid `ContextHelper`? So far as I know it's fine

![image](https://user-images.githubusercontent.com/8270914/106646137-52c4f180-655b-11eb-84a2-c80ee28a8680.png)

Reply:
@davidjohnoliver well, for my understanding we try to avoid it to access to the current `Activity` as it's usually assumed to be a single instance while it could change overtime ... and it could even not be set if accessed too early. But it's also known that we have no other choices so far to use it sometimes, and actually here as it's just a check I would assume it's valid case.

Maybe @jeromelaban could give more details?

Reply:
Turns out the ContextHelper is not used here anyway, so I removed it